rental.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'book'
+require_relative 'class_person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Good job so far but here Getters and Setters for @book and @person instance variables should be present so kindly add them to att_accessor with the date ✨🙏🏻